qa/tasks/cephfs/filesystem.py

```diff
@@ -713,6 +713,11 @@ def create(self):
     def run_client_payload(self, cmd):
         # avoid circular dep by importing here:
         from tasks.cephfs.fuse_mount import FuseMount
+
+        # Wait for mds.0 to reach up::active state before mounting
+        # cephfs client in run_client_payload()
+        self.wait_for_state('up:active', timeout=30, rank=0)
```

Comment:
```suggestion
        self.wait_for_daemons()
```

there may be more than one active MDS when this method is called in future places.

Reply:
default timeout for `wait_for_daemons()` is 120 seconds, so `wait_for_daemons(timeout=30)` should suffice

Reply:
Done, and will use the default `120 seconds`.

Reply:
Thanks @batrick @dparmar18